src/Symfony/Bridge/Twig/Extension/TranslationExtension.php

```diff
@@ -94,11 +94,19 @@ public function getTranslationNodeVisitor()
 
     public function trans($message, array $arguments = array(), $domain = null, $locale = null)
     {
+        if (null === $this->translator) {
+            return $message;
```

Comment:
we should still replace arguments IMO. Otherwise this does not work even for the case where the English message is the source